projects/rero/ng-core/src/lib/record/search/record-search.component.html

```diff
@@ -202,7 +203,7 @@ <h5 *ngIf="types.length == 1 && showLabel">
           </ng-core-record-search-aggregation>
         </div>
       </div>
-      <div id="recordlist" class="col">
+      <div id="recordlist" class="col-sm-12 col-md-7 col-lg-9 order-sm-1 order-md-2">
```

Comment:
I tested and think the following will be more appropriate:

```
<div id="recordlist" class="col-sm-12 col-md-7 col-lg-9 order-1 order-md-2">
```

(Same for the other column)